lib/config.py

```diff
@@ -145,7 +141,7 @@ def discover_packages():
     versions_repo_url = config.get('build_versions_repository_url')
     versions_repo_name = os.path.basename(os.path.splitext(versions_repo_url)[0])
     build_versions_repo_dir = os.path.join(
-        config.get('build_versions_repo_dir'),
+        config.get('work_dir'),
```

Comment:
Done, `versions` is now placed under <work-dir>/<REPOSITORIES_DIR> (REPOSITORIES_DIR is defined on lib/constants.py to avoid circular dependency)